src/lib/types.ts

```diff
@@ -174,9 +174,18 @@ export interface UserCredentials {
   refreshToken?: string
   // (Optional) The name of the provider.
   provider?: Provider
+  openIDConnect?: OpenIDConnectCredentials
```

Comment:
```suggestion
oidc?: OpenIDConnectCredentials
```

Consider abbreviation?